src/web/app/pom.xml

```diff
@@ -634,6 +634,21 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
+      <id>monitor-micrometer</id>
+      <activation>
+        <property>
+          <name>monitoring</name>
+        </property>
+      </activation>
```

Comment:
See above - If you are going to have an activation property here, you should have it in community/pom.xml also :)

Reply:
Thanks for the comment.
None of the monitoring extensions have an activation property in the `community/pom.xml`. Would you like me to add it specifically for this extension?

Reply:
The monitor extension is supported, so you'll find its activation property under extensions/pom.xml
The monitor-hibernate and monitor-kafka both have profiles in community/pom.xml:

```
    <profile>
      <id>monitor-hibernate</id>
      <modules>
        <module>monitor-hibernate</module>
      </modules>
    </profile>
    <profile>
      <id>monitor-kafka</id>
      <modules>
        <module>monitor-kafka</module>
      </modules>
    </profile>
```

Reply:
Ok I see now... the monitoring property is really a bad idea, when used it brings in all sorts of monitoring storages, but only one should be included instead. Better to remove the property altogether and just rely on direct profile activation with -P instead.

Reply:
Removed the activation properties to avoid load all the extensions at once.
Thanks for the comments.